core_integrations/iron_os/icon.txt

```diff
@@ -0,0 +1 @@
+mdi:soldering-iron
```

Comment:
This is not a helper or a Home Assistant base internal component. This this should not be using this MDI icon.

../Frenck

Reply:
Using the MDI icon was a suggestion by @joostlek as IronOS is not a brand and doesn't have a logo. I had suggested this but Joost just laughed 😬 ![image-1.png](https://github.com/user-attachments/assets/5c7cf2e4-12f2-4f4e-81d7-b55dba0db9c9)

Reply:
Oh, the soldering iron was suggested by Frenck

Reply:
What I meant with that, is just picking the raw icon image from their website.
Like this:

![light](https://github.com/user-attachments/assets/81aa6ced-99d4-4d74-8cb3-78309fa0e7c6)
![dark](https://github.com/user-attachments/assets/6ff5a932-4344-4fc4-92e8-3e90163412fb)